index.Rmd

```diff
@@ -40,79 +40,72 @@ can_lang <- readr::read_csv("https://github.com/ttimbers/canlang/raw/master/inst
 
-The name Jupyter came from combining the names of the three programming language that it was initially targeted for (Julia, Python, and R), and now many other languages can be used with Jupyter notebooks. 
-
-A notebook looks like this:
-
-```{r img-jupyter, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "Jupyter Notebook", fig.retina = 2}
-knitr::include_graphics("img/jupyter.png")
-```
-
-We have included a short demo video here to help you get started and to introduce you to R and Jupyter. 
-However, the best way to learn how to write and run code and formattable text in a Jupyter notebook is to do it yourself! [Here is a worksheet](https://github.com/UBC-DSCI/dsci-100-assets/blob/master/2019-fall/materials/worksheet_01/worksheet_01.ipynb) that provides a step-by-step guide through the basics.
-
-<iframe width="840" height="473" src="https://www.youtube.com/embed/iel3ZvS-Ryc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-
-## Loading a spreadsheet-like dataset
-
-Often, the first thing we need to do in data analysis is to load a dataset into R. When we bring spreadsheet-like (think Microsoft Excel tables) data, generally shaped like a rectangle, into R it is represented as what we call a *data frame* object. It is very similar to a spreadsheet where the rows are the collected observations and the columns are the variables. 
+Often, the first step in data analysis is to load a data set into R. When we bring spreadsheet-like (think Microsoft Excel tables) data, generally shaped like a rectangle, into R, it is represented as a *data frame* object. In figure \@ref(fig:img-spreadsheet-vs-dataframe), we can see an R data frame is very similar to a spreadsheet where the rows are the collected observations, and the columns are the variables. 
 
 
 ```{r img-spreadsheet-vs-dataframe, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "A spreadsheet versus a data frame in R", out.width="850", fig.retina = 2}
 knitr::include_graphics("img/spreadsheet_vs_dataframe.PNG")
 ```
 
-The first kind of data we will learn how to load into R (as a data frame) is the 
+The first kind of data file that we will learn how to load into R (as a data frame) is the 
 spreadsheet-like *comma-separated values* format (`.csv` for short).
-These files have names ending in `.csv`, and can be opened open and saved from common spreadsheet programs like Microsoft Excel and Google Sheets. 
+These files have names ending in `.csv`, and can be opened and saved using common spreadsheet programs like Microsoft Excel and Google Sheets. 
 For example, a `.csv` file named `can_lang.csv` [is included with the code for this book](https://github.com/UBC-DSCI/introduction-to-datascience/blob/master/data/can_lang.csv). 
 This file--- originally from [{canlang} R data package](https://ttimbers.github.io/canlang/)---has language data collected in the 2016 Canadian census [@cancensus2016]. 
```

Comment:
We should formally cite all R packages. The citation for this one is here: https://ttimbers.github.io/canlang/authors.html

Reply:
Good point! Will add!